src/Stratis.Bitcoin.Features.Wallet/WalletSettings.cs

```diff
@@ -22,38 +22,14 @@ public class WalletSettings
         /// </summary>
         public bool IsLightWallet { get; set; }
 
-        /// <summary>
-        /// A callback allow changing the default settings.
-        /// </summary>
-        private readonly Action<WalletSettings> callback;
-
-        /// <summary>
-        /// Initializes an instance of the object.
-        /// </summary>
-        public WalletSettings()
+        public WalletSettings() : this(NodeSettings.Default())
```

Comment:
Are we using this constructor?

Reply:
No. Was used in the tests before, forgot to remove it after fixing the tests

Reply:
Actually, this parameterless constructor is still needed for the tests- without it you can't mock the class

Reply:
Same for dns settings constructor

Reply:
ah yeah good point